app/templates/briefs/_brief_response_data.html

```diff
@@ -27,17 +27,19 @@
             {% elif item.type == "dynamic_list" %}
               {% for question in item.questions %}
                   {% if question.type == "boolean" %}
-                    {% if question._data['followup'] %}
+                    {% if question._data['values_followup'] %}
                       {% call summary.row() %}
                         {{ summary.field_name(question.question) }}
-                        {{ summary.text(item.unformat_data(brief_response)[question._data['followup']]) }}
+                        {{ summary.text(item.unformat_data(brief_response)[question._data['values_followup']]) }}
```

Comment:
I haven't realised followup was used here, but this change won't work. Old followup format stored a single string value that matched to follow-up question ID. Now both `.followup` and `.values_followup` are dictionaries, so we either need to either change this part to deal with multiple follow-ups or to make it always take the first one for now.